inst/Upgrades.qml

```diff
@@ -0,0 +1,287 @@
+import QtQuick		2.12
+import JASP.Module	1.0
+
+Upgrades
+{
+	Upgrade
+	{
+		functionName: 		"BinomialTest"
+		fromVersion:		"0.16.4"
+		toVersion:			"0.17.0"
+
+		ChangeRename{	from: "hypothesis";	to: "alternative"	}
+		ChangeJS
+		{
+			name:		"alternative"
+			jsFunction:	function(options)
+			{
+				switch(options["alternative"])
+				{
+					case "notEqualToTestValue":		return "twoSided";
+					case "greaterThanTestValue":	return "greater";
+					case "lessThanTestValue":		return "less";
+				}
+			}
+		}
+		ChangeRename{	from: "confidenceInterval";						to: "ci"						}
+		ChangeRename{	from: "confidenceIntervalInterval";				to: "ciLevel"					}
+		ChangeRename{	from: "VovkSellkeMPR";							to: "vovkSellke"				}
+		ChangeRename{	from: "descriptivesPlots";						to: "descriptivesPlot"			}
+		ChangeRename{	from: "descriptivesPlotsConfidenceInterval";	to: "descriptivesPlotCiLevel"	}
+	}
+
+	Upgrade
+	{
+		functionName: 		"BinomialTestBayesian"
+		fromVersion:		"0.16.4"
+		toVersion:			"0.17.0"
+
+		ChangeRename{	from: "hypothesis";	to: "alternative"	}
+		ChangeJS
+		{
+			name:		"alternative"
+			jsFunction:	function(options)
+			{
+				switch(options["alternative"])
+				{
+					case "notEqualToTestValue":		return "twoSided";
+					case "greaterThanTestValue":	return "greater";
+					case "lessThanTestValue":		return "less";
+				}
+			}
+		}
+		ChangeRename{	from: "plotPriorAndPosterior";					to: "priorPosteriorPlot"					}
+		ChangeRename{	from: "plotPriorAndPosteriorAdditionalInfo";	to: "priorPosteriorPlotAdditionalInfo"	}
+		ChangeRename{	from: "plotSequentialAnalysis";					to: "bfSequentialPlot"				}
+		ChangeRename{	from: "descriptivesPlots";						to: "descriptivesPlot"						}
+		ChangeRename{	from: "descriptivesPlotsCredibleInterval";		to: "descriptivesPlotCiLevel"				}
+	}
+
+	Upgrade
+	{
+		functionName:		"MultinomialTest"
+		fromVersion:		"0.16.4"
+		toVersion:			"0.17.0"
+
+		// this option did not do anything and was always hidden so now it's removed
+		ChangeRemove{	name: "simulatepval"	}
+		ChangeRename{	from: "counts";					to: "count"				}
+		ChangeRename{	from: "exProbVar";				to: "expectedCount"		}
+		ChangeRename{	from: "hypothesis";				to: "testValues"		}
```

Comment:
nullValues(?)

Reply:
We could go with `nullValues`, although it would need to be changed in the docs: https://github.com/jasp-stats/jasp-desktop/blob/stable/Docs/development/guide-option-names.md#list-of-common-option-names. We also use `testValue` already in other analyses, e.g., t-tests.

Reply:
It didn't occur to me that testValue referred to the values of the null hypothesis. But I don't want to stall this PR much longer and for the moment it's good.